composer.json

```diff
@@ -1,45 +1,47 @@
 {
-    "name": "league/openapi-psr7-validator",
-    "description": "Validate PSR-7 messages against OpenAPI (3.0.2) specifications \nexpressed in YAML or JSON",
-    "keywords": [
-        "OpenAPI",
-        "psr7",
-        "http",
-        "validation"
-    ],
-    "homepage": "https://github.com/thephpleague/openapi-psr7-validator",
-    "license": "MIT",
-    "type": "library",
-    "autoload": {
-        "psr-4": {
-            "League\\OpenAPIValidation\\": "src/"
-        }
-    },
-    "autoload-dev": {
-        "psr-4": {
-            "League\\OpenAPIValidation\\Tests\\": "tests/"
-        }
-    },
-    "require": {
-        "php": ">=7.2",
-        "ext-json": "*",
-        "cebe/php-openapi": "^1.3",
-        "dflydev/fig-cookies": "^2.0",
-        "psr/cache": "^1.0",
-        "psr/http-message": "^1.0",
-        "psr/http-server-middleware": "^1.0",
-        "respect/validation": "^1.1",
-        "riverline/multipart-parser": "^2.0.3",
-        "webmozart/assert": "^1.4"
-    },
-    "require-dev": {
-        "cache/array-adapter": "^1.0",
-        "doctrine/coding-standard": "^6.0",
-        "guzzlehttp/psr7": "^1.5",
-        "phpunit/phpunit": "^7|^8"
-    },
-    "config": {
-        "sort-packages": true
-    },
-    "prefer-stable": true
+  "name": "league/openapi-psr7-validator",
+  "description": "Validate PSR-7 messages against OpenAPI (3.0.2) specifications \nexpressed in YAML or JSON",
```

Comment:
@edmondscommerce any reason for reformatting the file?

Reply:
muscle memory I'm afraid - I always reformat before save

is it a big deal?

Reply:
I don't think so, the feature in this PR is way more important. :) Can we merge it?

Reply:
The problem is that since there is no `.editorconfig` in project - next contributor could reformat it back. It's not really a big problem but I'd rather suggest to keep meaningful changes only